www/js/main.js

```diff
@@ -4463,7 +4463,7 @@ function showOptions( expandItem ) {
 		page.find( "#o23" ).prop( "disabled", ( parseInt( this.value ) === 0 ? false : true ) );
 
 		// Switch the state of adjustment options based on the selected method
-		page.find( "#wto" ).parents( ".ui-field-contain" ).toggleClass( "hidden", parseInt( this.value ) === 0 ? true : false );
+		page.find( "#wto" ).click().parents( ".ui-field-contain" ).toggleClass( "hidden", parseInt( this.value ) === 0 ? true : false );
```

Comment:
Hey Samer, I see that this will open the new options popup. What happens if the user dismisses the popup i.e. clicks outside the window. Will App revert to the previous adjustment method or retain the new method and what would the options reflect in that event ?